PSL/SENSORS/Sx1276.py

```diff
@@ -0,0 +1,331 @@
+#Registers adapted from sample code for SEMTECH SX1276
+import time
+
+def connect(SPI,frq,**kwargs):
+	return SX1276(SPI,frq,**kwargs)
+
+
+class SX1276():
+	name = 'SX1276'
+	#registers
+	REG_FIFO                 = 0x00
+	REG_OP_MODE              = 0x01
+	REG_FRF_MSB              = 0x06
+	REG_FRF_MID              = 0x07
+	REG_FRF_LSB              = 0x08
+	REG_PA_CONFIG            = 0x09
+	REG_LNA                  = 0x0c
+	REG_FIFO_ADDR_PTR        = 0x0d
+	REG_FIFO_TX_BASE_ADDR    = 0x0e
+	REG_FIFO_RX_BASE_ADDR    = 0x0f
+	REG_FIFO_RX_CURRENT_ADDR = 0x10
+	REG_IRQ_FLAGS            = 0x12
+	REG_RX_NB_BYTES          = 0x13
+	REG_PKT_RSSI_VALUE       = 0x1a
+	REG_PKT_SNR_VALUE        = 0x1b
+	REG_MODEM_CONFIG_1       = 0x1d
+	REG_MODEM_CONFIG_2       = 0x1e
+	REG_PREAMBLE_MSB         = 0x20
+	REG_PREAMBLE_LSB         = 0x21
+	REG_PAYLOAD_LENGTH       = 0x22
+	REG_MODEM_CONFIG_3       = 0x26
+	REG_RSSI_WIDEBAND        = 0x2c
+	REG_DETECTION_OPTIMIZE   = 0x31
+	REG_DETECTION_THRESHOLD  = 0x37
+	REG_SYNC_WORD            = 0x39
+	REG_DIO_MAPPING_1        = 0x40
+	REG_VERSION              = 0x42
+	REG_PA_DAC				 = 0x4D
```

Comment:
That's superb! :D I was checking the device in eBay. There are two types I found [`SPI`](http://www.ebay.com/itm/RCmall-2-4G-Wireless-Wifi-SPI-Transceiver-Module-with-Shield-Case-22dBm-100mW/182501827021?_trksid=p2047675.c100005.m1851&_trkparms=aid%3D555018%26algo%3DPL.SIM%26ao%3D2%26asc%3D41376%26meid%3D16c81dc3d550445e8d80c0214596cfd3%26pid%3D100005%26rk%3D2%26rkt%3D6%26sd%3D192002707560) and [`TTL`](http://www.ebay.com/itm/E44-TTL-100-20dBm-long-range-3km-SX1276-SX1278-915MHz-LoRa-RF-Transceiver-Module-/192002707560?hash=item2cb440d068:g:81IAAOSwZQRYbz8O)! Current implementation is using `SPI` only right?